.github/workflows/check-standard.yaml

```diff
@@ -0,0 +1,80 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+name: R-CMD-check
```

Comment:
Again, this should be at the top and we need a better name for the workflow and the file.